packages/cactus-plugin-ledger-connector-iroha/src/main/typescript/plugin-ledger-connector-iroha.ts

```diff
@@ -0,0 +1,684 @@
+import { Server } from "http";
+import * as grpc from "grpc";
+import { Server as SecureServer } from "https";
+import { CommandService_v1Client as CommandService } from "iroha-helpers-ts/lib/proto/endpoint_grpc_pb";
+import { QueryService_v1Client as QueryService } from "iroha-helpers-ts/lib/proto/endpoint_grpc_pb";
+import commands from "iroha-helpers-ts/lib/commands/index";
+import queries from "iroha-helpers-ts/lib/queries";
+//import RolePermission from "iroha-helpers-ts/lib/proto/primitive_pb";
+import type { Express } from "express";
+import { promisify } from "util";
+import { Optional } from "typescript-optional";
+import {
+  GrantablePermission,
+  GrantablePermissionMap,
+} from "iroha-helpers-ts/lib/proto/primitive_pb";
+//import { Contract, ContractSendMethod } from "web3-eth-contract";
+
+import {
+  ConsensusAlgorithmFamily,
+  IPluginLedgerConnector,
+  IWebServiceEndpoint,
+  IPluginWebService,
+  ICactusPlugin,
+  ICactusPluginOptions,
+} from "@hyperledger/cactus-core-api";
+
+import {
+  PluginRegistry,
+  consensusHasTransactionFinality,
+} from "@hyperledger/cactus-core";
+
+import {
+  Checks,
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+  Http405NotAllowedError,
+} from "@hyperledger/cactus-common";
+
+import {
+  IrohaCommand,
+  IrohaQuery,
+  // InvokeContractV1Request,
```

Comment:
@hxlaf Same here, commented lines should just be deleted unless there's some specific need to use comments as a backup mechanism (usually we just use git's history for that but there are exceptoins where it's justifiable)

Reply:
Removed! Thank you!